packages/autocannon-ui-frontend/src/components/ResultSet.jsx

```diff
@@ -11,33 +11,40 @@ import {
   TableHead,
   TableRow,
   Paper,
-  Tooltip
+  Tooltip,
+  AccordionDetails
 } from '@material-ui/core'
 import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
 import { makeStyles } from '@material-ui/core/styles'
 import T from 'prop-types'
 import prettyBytes from 'pretty-bytes'
 import HelpIcon from '@material-ui/icons/Help'
 
-const useStyles = makeStyles({
+const useStyles = makeStyles(theme => ({
   table: {
     minWidth: 650,
-    padding: '30px'
+    padding: theme.spacing(3)
```

Comment:
just checking. 30px is closer to spacing(4) than spacing(3). I don't mind either way

Reply:
IMHO spacing(3) looked better and was more consistent with the paddings used elsewhere.